deployment/Makefile

```diff
@@ -0,0 +1,51 @@
+COMPONENT=
+STAGEPATH=${shell cd active && pwd -P}
+STAGE=${shell basename $(STAGEPATH)}
+DIRS=${shell find $(STAGE)/* -not -path "*/\.*" -type d}
+COMPONENTS=${shell basename $(DIRS)}
+AWS_PROFILE=${shell cat $(STAGE)/local_variables.tf | jq -r .variable.aws_profile.default}
```

Comment:
The AWS profile name should not be set from the terraform config. AWS profiles are details of the user's configuration that are orthogonal to terraform and should not be entangled.